.github/workflows/gem-push.yml

```diff
@@ -0,0 +1,32 @@
+name: Ruby Gem
+
+on:
+#   push:
+#     branches: [ master ]
+  # run manually for now
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build + Publish
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
```

Comment:
Not a required change, but I believe @npaufler set the default Actions permission to `read`, so I think we get `read` by default.